das/options.go

```diff
@@ -42,10 +42,10 @@ type Parameters struct {
 	// ConcurrencyLimit.
 	SampleTimeout time.Duration
 
-	// SamplingWindow determines the time window that headers should fall into
+	// samplingWindow determines the time window that headers should fall into
 	// in order to be sampled. If set to 0, the sampling window will include
 	// all headers.
-	SamplingWindow time.Duration
+	samplingWindow time.Duration
```

Comment:
Perhaps changing samplingWindow type from time.Duration to pruner.AvailabilityWindow will simplify code a lot. I assume it was time.duration to avoid importing `pruner` as a dependency. But the dependancy inevitable, considering DASer has to rely on pruner logic to skip pruned heights. Using native type for window here can remove the need to cast it everywhere and for cast method itself.

Reply:
just did

Reply:
also in archival PR, daser will use pruner convenience method to check recency instead of its own